isis/src/base/objs/History/History.h

```diff
@@ -29,27 +29,24 @@ namespace Isis {
    *                           include for Pvl since the include for Pvl was
    *                           removed from Blob.h. Added padding to control
    *                           statements. References #1169
-   *  
+   *
    * @todo This class needs documentation.
   */
-  class History : public Isis::Blob {
+  class History {
     public:
-      History(const QString &name);
-      History(const QString &name, const QString &file);
+      History();
+      History(Isis::Blob &blob);
       ~History();
 
       void AddEntry();
       void AddEntry(Isis::PvlObject &obj);
       Pvl ReturnHist();
-      void Read(const Isis::Pvl &pvl, std::istream &is);
 
-    protected:
-      void WriteInit();
+      Blob *toBlob(const QString &name = "IsisCube");
```

Comment:
We may want to change this to just return a char *buffer and let the blob class construct itself from it